ko/mavgen_c/example_c_udp.md

```diff
@@ -1,9 +1,9 @@
 # MAVLink C UDP Example
```

Comment:
Can you please undo changes to this file and the zh path file. It isn't wrong, but the fixes round trip through our translation software, and if there are clashes it is a pain to merge them.

Reply:
I've already reverted the changes to the non-English files (`zh` and `ko`) and kept only the updates to the English version (`en`). This should prevent any issues when processing through the translation software. Let me know if anything else is needed!